packages/client/manifest.json

```diff
@@ -6339,6 +6342,18 @@
         "label": "High contrast",
         "key": "stripeRows",
         "defaultValue": false
+      },
+      {
+        "section": true,
+        "name": "Columns",
+        "settings": [
+          {
+            "type": "columns/grid",
+            "key": "columns",
+            "nested": true,
+            "resetOn": "dataSource"
```

Comment:
Shouldn't this be `resetOn: "table"` as thats this key for the target field? `dataSource` is the type of field